shared/js/media/jpeg_metadata_parser.js

```diff
@@ -20,6 +20,10 @@ function parseJPEGMetadata(file, metadataCallback, metadataError) {
   // This is the object we'll pass to metadataCallback
   var metadata = {};
 
+  // this is the start offset for the data in exif. Should be
+  // where to find the TIFF magic header.
+  var startOffset = 0;
```

Comment:
TIFFHeaderOffset it is. I don't agree with the const. For the same argument as "+ 10".